app/src/test/java/ca/klapstein/baudit/data/PasswordTest.java

```diff
@@ -4,33 +4,82 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import static junit.framework.TestCase.fail;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
 
 public class PasswordTest {
-    @Before
-    public void setUp() {
+    @Test
+    public void test_validPassword() {
+        assertTrue(Password.isValid("abcd1234"));
+        assertTrue(Password.isValid("password"));
+        assertTrue(Password.isValid("011235813"));
     }
 
-    @After
-    public void tearDown() {
+    @Test
+    public void test_invalidPassword() {
```

Comment:
these are good tests!

Though you could parameterize them with junit:

https://github.com/junit-team/junit4/wiki/parameterized-tests

That would be really dope